tests/cpp/test_memory.cpp

```diff
@@ -1496,80 +1496,6 @@ TEST_F(TMARuntimeInvalidTest, InvalidView) {
           ::testing::HasSubstr("Invalid view in TMA: the extent of")));
 }
 
-TEST_F(TMACompileTimeInvalidTest, DependentBoxingSplit1) {
```

Comment:
This test is removed. Because it is failing, and I don't think it makes sense anymore. For this test, we can consider the tensor as view `(-1. 16, 16)` and define box on this view. This is totally a valid case.